src/theory/tools/wp/bir_htScript.sml

```diff
@@ -0,0 +1,614 @@
+open HolKernel Parse boolLib bossLib;
+
+(* From /core: *)
+open bir_programTheory bir_typing_progTheory bir_envTheory
+     bir_auxiliaryTheory bir_valuesTheory bir_expTheory
+     bir_exp_immTheory bir_typing_expTheory bir_immTheory;
+
+(* From /theories: *)
+open bir_program_blocksTheory bir_program_terminationTheory
+     bir_program_valid_stateTheory bir_exp_substitutionsTheory
+     bir_bool_expTheory bir_program_env_orderTheory
+     bir_program_multistep_propsTheory bir_exp_equivTheory
+     bir_subprogramTheory;
+
+open bir_wpTheory;
+
+open HolBACoreSimps;
+
+val _ = new_theory "bir_ht";
```

Comment:
Wouldn't `bir_hoare_triple` be easier to understand? Or you can at least add a comment in the top of the file to explain the acronym :)

Reply:
good idea, I think a comment suffices.

Reply:
"HT" is a very well established acronym for "Hoare triple", just like "WP" is for "weakest precondition", so I think this is OK. But I will add a comment describing the theory (IMO good practice anyway).

Reply:
Oh, I didn't know about it. It's fine then :)